client-keystore/src/lib.rs

```diff
@@ -36,6 +35,10 @@ use std::{
 
 use sc_keystore::{Error, Result};
 
+mod keystore_ext;
+pub use keystore_ext::KeystoreExt;
```

Comment:
Do we need this line? The KeystoreExt is not used here, right?

Reply:
Yes, otherwise the `KeystoreExt` won't be accessible from the outside.
We could do it the following way:
```rust
pub mod keystore_ext;
```
but then one would need to import it with `use substrate_client_keystore::keystore_ext::KeystoreExt;`. I prefer the above TBH